src/pages/myPage.js

```diff
@@ -1,7 +1,217 @@
-import React from 'react';
-
-const MyPage = () => {
-  return <div>MyPage</div>;
-};
-
-export default MyPage;
+import React, { useCallback, useEffect, useState, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { removeCookie } from '../components/Shared/Cookies';
+import { getUserInfo } from '../store/myPage';
+import SideNavbar from 'components/SideNavbar';
+import LoginButton from 'components/Shared/LoginButton';
+import EditNickName from 'components/Mypage/EditNickName';
+import SchoolAuth from 'components/Mypage/SchoolAuth';
+import AutoLogin from 'components/Mypage/AutoLogin';
+import styled from 'styled-components';
+
+const MyPage = () => {
+  const toggle = useSelector((state) => state.toggle.isOpen); //사이드 네비게이션 상태 전역 상태 관리
+  const userInformation = useSelector((state) => state.myPage); //2.저장된 유저 데이터 불러온다.
+  const loginInfo = useSelector((state) => state.auth);
+  const [userImgFile, setUserImgFile] = useState(''); //기본 이미지 url 유저 정보안에 들어있어야함
+
+  const setFile = (e) => {
+    if (e.target.files[0]) {
+      const img = new FormData();
+      img.append('file', e.target.files[0]);
+      axios
+        .post('', img)
+        .then((res) => {
+          setImageUrl(res.data);
+        })
+        .catch((err) => {
+          console.error(err);
+        });
+    }
+  };
+  const photoInput = useRef();
+  const [isShown, setIsShown] = useState(false);
+  const dispatch = useDispatch();
+  const naviagate = useNavigate();
+  /*
+    TODO:
+    [] 유저정보 받아올 것: 이미지, 닉네임, 학교인증여부  
+  */
+  const patch = () => {
+    photoInput.current.click();
+  };
+
+  const logoutClick = useCallback(() => {

```

Comment:
logoutClick, resignClick 은 어감이 이상하게 느껴집니다
단순히 리뷰 해결을 위해 수정하는 것이 아니라, 네이밍에 대해 정말 생각하고 만들어주세요
네이밍 관련하여 이전 PR 들이랑 별로 달라진게 없어 보입니다

```suggestion
  const clickLogoutButton = useCallback(() => {

```

```suggestion
  const logout = useCallback(() => {

```

Reply:
네이밍;;;

Reply:
나머지두;;;